src/main/kotlin/racingcar/domain/result/CarRaceResult.kt

```diff
@@ -0,0 +1,29 @@
+package racingcar.domain.result
+
+import racingcar.domain.car.Car
+
+class CarRaceResult(val carRaceResults: List<CarRaceResultForRound>) {
+    fun findWinners(): List<CarSnapShot> {
+        return carRaceResults.last().getWinnersForRound()
+    }
+}
+
+class CarRaceResultForRound(cars: List<Car>) {
+    val carSnapShots = cars.map { CarSnapShot(it) }
+
+    internal fun getWinnersForRound(): List<CarSnapShot> {
+        val maxPosition = carSnapShots.map { it.position }
+            .maxOf { it }
```

Comment:
```suggestion
        val maxPosition = carSnapShots.maxOf { it.position }
```

maxOf 만으로도 표현이 가능해보여요!